doc/en/reference/customize.rst

```diff
@@ -79,7 +79,7 @@ and can also be used to hold pytest configuration if they have a ``[pytest]`` se
 .. code-block:: ini
 
     # tox.ini
-    [pytest]
+    [tool:pytest]
```

Comment:
This doesn't seem right, AFAIK for `tox.ini` files we use the `[pytest]` section.